modules/custom/az_event/az_event_trellis/config/install/migrate_plus.migration.az_trellis_events.yml

```
@@ -29,54 +35,48 @@ source:
       selector: Summary
     -
       name: owner
-      label: Owner NetID
+      label: 'Owner NetID'
       selector: Owner_NetID
     -
       name: timezone
-      label: Time Zone
+      label: 'Time Zone'
       selector: tz_name
     -
       name: start_date_time
-      label: Start Date and Time
+      label: 'Start Date and Time'
       selector: Event_Start_DateTime
     -
       name: end_date_time
-      label: End Date and Time
+      label: 'End Date and Time'
       selector: Event_End_DateTime
     -
       name: external_visibility
-      label: External Visibility
+      label: 'External Visibility'
       selector: External_Visibility
     -
       name: publish
       label: Publish
       selector: Publish
     -
       name: status_formula
-      label: Status Formula
+      label: 'Status Formula'
       selector: Status_Formula
   ids:
     id:
       type: string
-
 process:
   title: name
   field_az_trellis_id: id
   field_az_link/uri: url
   field_az_link/title:
     -
       plugin: default_value
-      default_value: Event details
+      default_value: 'Event details'
   field_az_summary:
     -
       plugin: skip_on_empty
       method: process
       source: summary
-
-  # Events are considered "published" unless any of these conditions are met:
-  # - 'External_Visibility' is "Don't Broadcast to any Platform (invite only)"
-  # - 'Status_Formula' is "Canceled"
-  # = 'Publish' is "false"
   external_visibility:
```

I think we should preserve these comments.
```suggestion

  # Events are considered "published" unless any of these conditions are met:
  # - 'External_Visibility' is "Don't Broadcast to any Platform (invite only)"
  # - 'Status_Formula' is "Canceled"
  # = 'Publish' is "false"
  external_visibility:
```